controller.lua

```diff
@@ -182,41 +182,18 @@ end
 local function validate_iid(iid)
 	if not iid then return true end -- nil is OK
```

Comment:
Need to check specifically for nil here:
```suggestion
	if iid == nil then return true end -- nil is OK
```